master/security/protect-hosts.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Protect hosts
+---
+
+### Big picture
+
+Use Calico network policy to restrict traffic to/from hosts.
+
+### Value
+
+Restricting traffic between hosts and the outside world is not unique to Calico; many solutions provide this capability. However, the advantage of using Calico to protect the host, it you can use the same Calico policy configuration as workloads. You only need to learn one tool. Write a cluster-wide policy, and it is immediately applied to every host. 
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+- **HostEndpoint** resource 
+- **GlobalNetworkPolicy**
+- **FelixConfiguration** resource with **FailsafeInboundHostPorts** and **DefaultEndpointToHostAction** parameters
+
+### Concepts
+
+#### Hosts and workloads
+
+In the context of Calico configuration, a **workload** is a virtualized compute instance, like a VM or container. A **host** is the computer that runs the hypervisor (for VMs), or container runtime (for containers). We say it “hosts” the workloads as guests.
+
+#### Host endpoints
+
+Each host has one or more network interfaces that it uses to communicate externally. You can use Calico network policy to secure these interfaces (called host endpoints). Calico network policy for hosts can have labels, and they work the same as labels on workload endpoints. The security rules are flexible so workloads and host endpoints can refer to each other using label selectors. 
+
+#### Failsafe rules
+
+It is easy to inadvertently cut all host connectivity because of non-existent or misconfigured network policy. To avoid this, Calico provides failsafe rules with default/configurable ports that are open on all host endpoints.
+
+#### Default behavior of workload to host traffic
+
+By default, Calico blocks all connections from a workload to its local host.You can control whether connections from a workload endpoint to its local host are dropped, returned, or accepted using a simple parameter.
```

Comment:
```suggestion
By default, Calico blocks all connections from a workload to its local host. You can control whether connections from a workload endpoint to its local host are dropped, returned, or accepted using a simple parameter.
```

Reply:
Fixed.

Reply:
This doesn't look fixed to me - there is still a missing space. @bmckercher123 have you pushed all of your commits for this one yet?

Reply:
> workload to its local host.You can control

Missing a space after the punctuation here

Reply:
Fixed.

Reply:
Fixed.